### DISCUSSION TITLE
Enhance search functionality with fuzzy matching

### DISCUSSION THREAD

**Discussion by walkerford** [memser-spaceport/pl-network-hub#6] [Ideas]
The current search function is very limited.  It only matches exact strings starting at the beginning of the name.

You cannot search by last name.  And you cannot search by approximate spelling.  

If I search for "Mona", nothing will come up, because the project is listed as "The Mona".

**Comment by pandeyanujk** [comment]
Thanks for submitting this request @walkerford. We have plans to improve the search. Your submissions help us prioritize things better.
